uc.c

```diff
@@ -230,6 +230,11 @@ static uc_err uc_init(uc_engine *uc)
         return UC_ERR_RESOURCE;
     }
 
+    // init tlb function
+    if (!uc->cpu->cc->tlb_fill) {
+        uc->set_tlb(uc, UC_TLB_VIRTUAL);
```

Comment:
But you are setting `UC_TLB_VIRTUAL` as default, no?

Reply:
Yes I understand your reasoning here. However, with MMU disabled, `vaddr=paddr` is satisfied in most cases, yes? If so, we can still preserve the `TLB_CPU` implementation by default.

Reply:
> However, with MMU disabled, vaddr=paddr is satisfied in most cases, yes?

Most cases yes, but the biggest problem I see is x86_64. By default only a part off the address space is usable. But when you prefere TLB_CPU I can change it.